src/math_util_2.c

```diff
@@ -342,6 +342,7 @@ Vec3f *vec3f_set_dupe_2_electric_boogaloo(Vec3f arg0, f32 arg1, f32 arg2, f32 ar
     return (Vec3f *) &arg0;
 }
 
+// look like normalise vector
 Vec3f *func_80041530(Vec3f dest) {
```

Comment:
Please name this func `normalize_vector` (please check there isn't already a func named `normalize_vector`)

// @warning Returns addr of local variable.

Reply:
you have also func_802B6374 who do the same thing

Reply:
Okay, this is fine then.